datasets/fr/tresor/fr_tresor_gels_avoir.yml

```
@@ -4182,3 +4182,12 @@ lookups:
         props:
           jurisdiction: ir
           incorporationDate: 2014
+      - match: "Type d’entité: gang haïtien opérant principalement dans les départements de l’Artibonite et du Nord-Ouest: /"
+        props:
+          legalForm: gang haïtien
+          address: "départements de l’Artibonite et du Nord-Ouest"
+      - match: "Type d’entité: centre de rétention - Lieu d’enregistrement: Comté d’Onsong, province de Hamgyong du Nord, République populaire démocratique de Corée (RPDC): /"
+        props:
+          jurisdiction: kp
+          legalForm: centre de rétention
```

```suggestion
          description: centre de rétention
```

here as well, detention centre doesn't fit so well in legal form. Other cases have values like limited liability company. I'd say we put it in description.

---

not 100% sure it'll accept the description prop name here, but let's see?